recipe/meta.yaml

```diff
@@ -19,13 +19,12 @@ requirements:
   host:
```

Comment:
I see `requires-python = ">=3.8"`, so we should skip # [py<**38**].

Reply:
Done